import ofxCv.h give error: ‘cv::Size2d’ has not been declared

I'm using the last version of master branch of openFramework and the last version of develop branch of ofxCv.
I'm using GNU Make 3.81 under linux Mint 17.1 64bit.

Compiling the project "example-face" give me these errors:
<pre>
In file included from /home/giulio/git/openFrameworks/addons/ofxCv/src/ofxCv.h:22:0,
                 from /home/giulio/git/openFrameworks/addons/ofxCv/example-face/src/ofApp.h:4,
                 from /home/giulio/git/openFrameworks/addons/ofxCv/example-face/src/ofApp.cpp:1:
/home/giulio/git/openFrameworks/addons/ofxCv/libs/ofxCv/include/ofxCv/Calibration.h:30:67: error: ‘cv::Size2d’ has not been declared
         void setup(float focalLengthMm, cv::Size imageSizePx, cv::Size2d sensorSizeMm, cv::Point2d principalPointPct = cv::Point2d(.5,.5));
                                                                   ^
In file included from /home/giulio/git/openFrameworks/addons/ofxCv/src/ofxCv.h:22:0,
                 from /home/giulio/git/openFrameworks/addons/ofxCv/example-face/src/ofApp.h:4,
                 from /home/giulio/git/openFrameworks/addons/ofxCv/example-face/src/ofApp.cpp:1:
/home/giulio/git/openFrameworks/addons/ofxCv/libs/ofxCv/include/ofxCv/Calibration.h:31:58: error: ‘cv::Size2d’ has not been declared
   void setup(Mat cameraMatrix, cv::Size imageSizePx, cv::Size2d sensorSizeMm = cv::Size2d(0, 0));
                                                          ^
/home/giulio/git/openFrameworks/addons/ofxCv/libs/ofxCv/include/ofxCv/Calibration.h:35:3: error: ‘Size2d’ in namespace ‘cv’ does not name a type
   cv::Size2d getSensorSize() const;
   ^
/home/giulio/git/openFrameworks/addons/ofxCv/libs/ofxCv/include/ofxCv/Calibration.h:45:9: error: ‘Size2d’ in namespace ‘cv’ does not name a type
         cv::Size2d sensorSize;
         ^
/home/giulio/git/openFrameworks/addons/ofxCv/libs/ofxCv/include/ofxCv/Calibration.h:31:80: error: ‘Size2d’ is not a member of ‘cv’
   void setup(Mat cameraMatrix, cv::Size imageSizePx, cv::Size2d sensorSizeMm = cv::Size2d(0, 0));
</pre>

--- a/libs/ofxCv/include/ofxCv/Calibration.h
+++ b/libs/ofxCv/include/ofxCv/Calibration.h
@@ -27,12 +27,12 @@ namespace ofxCv {
 	
 	class Intrinsics {
 	public:
-        void setup(float focalLengthMm, cv::Size imageSizePx, cv::Size2d sensorSizeMm, cv::Point2d principalPointPct = cv::Point2d(.5,.5));
-		void setup(Mat cameraMatrix, cv::Size imageSizePx, cv::Size2d sensorSizeMm = cv::Size2d(0, 0));
+        void setup(float focalLengthMm, cv::Size imageSizePx, cv::Size2f sensorSizeMm, cv::Point2d principalPointPct = cv::Point2d(.5,.5));
+		void setup(Mat cameraMatrix, cv::Size imageSizePx, cv::Size2f sensorSizeMm = cv::Size2f(0, 0));
 		void setImageSize(cv::Size imgSize);
 		Mat getCameraMatrix() const;
 		cv::Size getImageSize() const;
-		cv::Size2d getSensorSize() const;
+		cv::Size2f getSensorSize() const;
 		cv::Point2d getFov() const;
 		double getFocalLength() const;
 		double getAspectRatio() const;
@@ -42,7 +42,7 @@ namespace ofxCv {
         void updateValues();
 		Mat cameraMatrix;
         cv::Size imageSize;
-        cv::Size2d sensorSize;
+        cv::Size2f sensorSize;
 		cv::Point2d fov;
 		double focalLength, aspectRatio;
 		Point2d principalPoint;

--- a/libs/ofxCv/src/Calibration.cpp
+++ b/libs/ofxCv/src/Calibration.cpp
@@ -6,7 +6,7 @@ namespace ofxCv {
     
     using namespace cv;
     
-    void Intrinsics::setup(float focalLength, cv::Size imageSize, cv::Size2d sensorSize, cv::Point2d principalPoint) {
+    void Intrinsics::setup(float focalLength, cv::Size imageSize, cv::Size2f sensorSize, cv::Point2d principalPoint) {
         float focalPixels = (focalLength / sensorSize.width) * imageSize.width;
         float fx = focalPixels; // focal length in pixels on x
         float fy = focalPixels;  // focal length in pixels on y
@@ -18,7 +18,7 @@ namespace ofxCv {
                           0, 0, 1);
         setup(cameraMatrix, imageSize, sensorSize);
     }
-    void Intrinsics::setup(Mat cameraMatrix, cv::Size imageSize, cv::Size2d sensorSize) {
+    void Intrinsics::setup(Mat cameraMatrix, cv::Size imageSize, cv::Size2f sensorSize) {
         this->cameraMatrix = cameraMatrix;
         this->imageSize = imageSize;
         this->sensorSize = sensorSize;
@@ -47,7 +47,7 @@ namespace ofxCv {
         return imageSize;
     }
     
-    cv::Size2d Intrinsics::getSensorSize() const {
+    cv::Size2f Intrinsics::getSensorSize() const {
         return sensorSize;
     }
     
@@ -131,7 +131,7 @@ namespace ofxCv {
         imagePoints.clear();
         FileStorage fs(ofToDataPath(filename, absolute), FileStorage::READ);
         cv::Size imageSize;
-        cv::Size2d sensorSize;
+        cv::Size2f sensorSize;
         Mat cameraMatrix;
         fs["cameraMatrix"] >> cameraMatrix;
         fs["imageSize_width"] >> imageSize.width;

